exercise/staff_views.py

```diff
@@ -179,7 +179,12 @@ class AllResultsView(CourseInstanceBaseView):
     def get_common_objects(self):
         super().get_common_objects()
         self.table = ResultTable(self.instance)
-        self.note("table")
+        self.tags = list(self.instance.usertags.all())
```

Comment:
Follow #391 for how to do tags better now. Internal and external are handled via the same interface now.

Reply:
Changed the tags to include  USERTAG_EXTERNAL, USERTAG_INTERNAL.